modules/editor/inserter/index.js

```diff
@@ -0,0 +1,22 @@
+const h = wp.element.createElement;
```

Comment:
I'd prefer something like `el` as well, but worth noting that `h` was turned into something of a convention with it also being adopted by [`preact`](https://github.com/developit/preact).